### ISSUE TITLE
Option to auto-save gif after recording

### ISSUE THREAD

**Issue by vadi2** [phw/peek#201]
Would it be possible to have Peek automatically start the render after capture, saving it using the default filepath it already uses?

**Comment by phw** [comment]
Yes, I already considered this but never actually implemented it. Needs some way to handle duplicate names (e.g. just append `(1)` or so). Not high priority, but patches welcome :)

**Comment by KKB33** [comment]
Good brother you can get new ui

**Comment by KKB33** [comment]
👍

**Comment by KKB33** [comment]
Trying
